misk/web/tabs/example/package.json

```diff
@@ -0,0 +1,28 @@
+{
+  "name": "misktab-example",
+  "version": "0.0.2",
```

Comment:
is this field necessary?

Reply:
Considering leaving this one in so that if there are breaking changes in the Example Tab that devs build off of, it can be seen which version they started with.